1-js/06-advanced-functions/02-rest-parameters-spread-operator/02-rest-operator-in-object-destructuring/task.md

```diff
@@ -0,0 +1,23 @@
+importance: 3
+
+---
+
+# Use rest operator with destructuring
+
+Let's say we have this object 'user':
+
+```js no-beautify
+let user = {
+	name: 'John',
+	lastname: 'Smith',
+	age: 32,
+	isAdmin: false,
+}
+```
+
+How could you use the rest operator with destructuring assignment in order to do this:
```

Comment:
Please write a destructuring operator to put:
- `user.name` ->  into `name` variable
- `user.lastname` -> into `lastname` variable
- the rest of object properties into `other` variable